impl/src/main/java/com/okta/sdk/impl/oauth2/AccessTokenRetrieverServiceImpl.java

```diff
@@ -125,26 +125,25 @@ public OAuth2AccessToken getOAuth2AccessToken() throws IOException, InvalidKeyEx
     /**
      * Create signed JWT string with the supplied token client configuration details.
      *
+     * @param jwtExpiryTime seconds to expiration of token. {@link #DEFAULT_EXPIRATION_OFFSET} will be used if not set.
      * @return signed JWT string
      * @throws InvalidKeyException
      * @throws IOException
      */
-    String createSignedJWT() throws InvalidKeyException, IOException {
+    String createSignedJWT(long jwtExpiryTime) throws InvalidKeyException, IOException {
```

Comment:
Set this value once in the constructor, read it from ClientConfiguration (or similar)